src/ir_Technibel.cpp

```diff
@@ -435,7 +403,7 @@ String IRTechnibelAc::toString(void) {
   String result = "";
   result.reserve(100);  // Reserve some heap for the string to reduce fragging.
   result += addBoolToString(getPower(), kPowerStr, false);
-  result += addModeToString(getMode(), kTechnibelAcCool, kTechnibelAcCool,
+  result += addModeToString(getMode(), 255, kTechnibelAcCool,
```

Comment:
What is this magic?

Reply:
There is no "Auto" mode, so I have to use an impossible value to make `addModeToString()` not display Auto.
Comment added.

I thought about adding one at the time. Thanks for pushing me. ;-)